test/test_integration.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import requests
+
+
+class TestIntegration:
+    def test_requests_integration(self):
+        with pytest.warns(DeprecationWarning) as record:
```

Comment:
Can we move this under `with_dummyserver` and make an http and https request to avoid a real network call?

Reply:
I've created a new file `test_integration.py` under `with_dummyserver` and added tests for http and https